src/commands/ImportTrialAppUriHandler.ts

```diff
@@ -0,0 +1,22 @@
+import * as querystring from 'querystring';
+import { Uri, UriHandler } from 'vscode';
+import { callWithTelemetryAndErrorHandling, IActionContext } from '../../extension.bundle';
+import { localize } from '../localize';
+import { importTrialApp } from './trialApp/importTrialApp';
+
+export class ImportTrialAppUriHandler implements UriHandler {
```

Comment:
Write a comment about how the uriHandler only activates when the uri has our extension id in it.  Should also link this https://code.visualstudio.com/api/references/activation-events#onUri